components/eam/src/physics/cam/zm_conv.F90

```diff
@@ -635,6 +642,9 @@ subroutine zm_convr(lchnk   ,ncol    , &
    real(r8), parameter :: mucon = 5.3_r8
    real(r8) negadq
    logical doliq
+
+   integer dcapemx(pcols)  ! maxi saved from 1st call for CAPE calculation;  used in 2nd call when DCAPE-ULL active
```

Comment:
Can you edit this to clarify what "maxi" is?

Reply:
"maxi" is replaced by "launching level index".